src/sentry/static/sentry/app/views/projectDashboard.jsx

```diff
@@ -152,7 +152,11 @@ const ProjectDashboard = createReactClass({
           </div>
           <h3>{t('Overview')}</h3>
         </div>
-        <ProjectChart dateSince={dateSince} resolution={resolution} />
+        <ProjectChart
+          dateSince={dateSince}
+          resolution={resolution}
+          environment={this.props.environment}
```

Comment:
Am I missing something, or is `environment` not a declared prop for `ProjectDashboard`? (It is accessed via `context` on [L98](https://github.com/getsentry/sentry/pull/7354/files#diff-abb03cf8790950f418bd32d97abc5a91L98) though.)

Reply:
This depends on the changes in https://github.com/getsentry/sentry/pull/7352

Reply:
@benvinegar Rebased, please take a look